minify.js

```diff
@@ -260,7 +260,7 @@ module.exports = function express_minify(options) {
                 res.setHeader('Content-Type', 'text/css');
             } else if (coffee_match.test(contentType)) {
                 type = TYPE_COFFEE;
-                res.setHeader('Content-Type', 'text/javascript');
+                res.setHeader('Content-Type', 'application/javascript');
```

Comment:
I don't think so, `text/javascript` is before standard, `application/javascript` is by standard.